docs/concepts.rst

```diff
@@ -1,128 +1,130 @@
+ 
+**Managed packages** are typically used by Salesforce partners to distribute and sell applications to customers. They are proprietary code that can only be upgraded and deployed by the developer that built them. To ensure seamless upgrades, certain destructive changes, like removing objects or fields, cannot be performed.
 
-Throughout the CumulusCI documentation, we'll have the Product Delivery Model in mind. 
+In CumulusCI, packages are also built, stored and deployed via **projects**.
 
 Projects
 --------
 
-When you work with CumulusCI, you do so inside a project. A project is a version control repository that contains Salesforce metadata as well as the CumulusCI automation that builds and releases the project. A project usually has a one-to-one relationship with a managed package, if building a package, or with an org implementation. A project may constitute the entirety of a product in the Product Delivery Model, or may be one of multiple projects making up a product.
+When you work with CumulusCI, you do so inside a project. A project is an individual git repository (think GitHub) that contains Salesforce metadata as well as the CumulusCI automation (e.g., tasks and flows) that builds and releases the project. Most importantly, a project can build one - and only one(!) - package in its repository. This one-to-one relationship between project and package is what allows version control to track any changes your team made to the repository with minimal collision.
 
-CumulusCI scopes many of its activities to the project, so you'll always run your CumulusCI commands inside your repository directory. Each project gets its own keychain, which we'll talk about in detail below. Each project has its own set of scratch orgs, which CumulusCI namespaces in the Salesforce CLI keychain to prevent collisions. And each project is linked to a single GitHub repository, so you can always be confident of the scope of the actions you're taking.
-
-CumulusCI knows which project you are working on by the current working directory of your shell. Make sure to change directories inside your project before you run project-specific commands.
-
-In order to be used as a CumulusCI project, a directory must both be a Git repository and contain a ``cumulusci.yml`` configuration file. We'll show you how to get set up with a new or existing CumulusCI project in the (TODO link) Get Started section.
+It's important to note that a project may not even contain a package; for example, it can be built to deliver functionality to records, or test data for QA. A project may constitute the entirety of a product made for release, or may be one of multiple projects banded together to make a product. Basically, whatever feature in your product that needs specialized Salesforce metadata and CumulusCI automation in order to be built and deployed, *especially* packages (we cannot stress this enough!), requires its own unique project. 
```

Comment:
```suggestion
A project doesn't have to contain a package. For example, it can deliver functionality to records or provide test data for QA. A project can constitute an entire product for release, or be just one of multiple projects banded together to make a product. Here's a rule of thumb for when a project _is_ required: whenever the thing that you're building and deploying requires specialized Salesforce metadata and CumulusCI automation, and *especially* packages. We can't stress this enough! 
```

Reply:
Will definitely strip out those mays/mights, but I need to advocate here for my original text because it adds more emphasis where needed. Saying "a project doesn't have to contain a package" is succinct, but it _is_ important to note because it's the reason for this paragraph's existence.  Because projects are not just vehicles for packages. (I made this assumption several times; @Br4nd0R corrected me each time.) Also, what's being a stressed enough (at the end) isn't the fact that a project only requires SF metadata and cci automation, which had just been clarified, but that **packages** definitely require a project because they're the bread and butter of cci.

My proposed edit:

"It's important to note that a project doesn't have to contain a package. For example, it can deliver functionality to records or provide test data for QA. A project can constitute an entire product for release, or be just one of multiple projects that together constitute a product. Essentially, if whatever you're building and deploying requires specialized Salesforce metadata and CumulusCI automation, *especially* packages (we cannot stress this enough!), it requires its own unique project."

(I also prefer "cannot" over "can't" in this instance because read aloud it emphasizes the stress.)

Reply:
@justinvholt Looks like we've landed in a great succinct place with David Reed's edits below and your concluding "To sum up." Yay!